.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+on:
+  pull_request:
+    branches: [ "master" ]
+jobs:
+  test:
+    name: test (JDK ${{ matrix.java }})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '11' ]
```

Comment:
It may be worth testing Java 8 because it is actively used in legacy web servers.